dist/eModal.js

```diff
@@ -452,12 +452,22 @@
             }
 
             var html = ('<div class=modal-body style="position: absolute;width: 100%;background-color: rgba(255,255,255,0.8);height: 100%;">%1%</div>' +
-                '<iframe class="embed-responsive-item" frameborder=0 src="%0%" style="width:100%;height:75vh;display:block;"/>')
+                    '<iframe class="embed-responsive-item" frameborder=0 src="%0%" style="width:100%;height:75vh;display:block;"/>')
                 .replace('%0%', params.message || params.url || params)
                 .replace('%1%', defaultSettings.loadingHtml);
 
-            var message = $(html)
-                .load(iframeReady);
+            var message;
+            if (jQuery.fn.jquery.split('.')
```

Comment:
We are accessing the global scope were! Please use the provided reference to jQuery